content/departments/tech-ops/tools/Okta/main.md

```diff
@@ -132,77 +100,91 @@ Apple Touch ID is authenticated through your browser, so if you use multiple bro
 
 ## Use a different MFA option
 
-To change between MFA options click on the arrow next to the label when prompted for MFA
+To change between MFA options, click on the arrow next to the label when prompted for MFA
 
 <img src="https://storage.googleapis.com/sourcegraph-assets/switch%20between.png" alt="Okta MFA options" width="300" height="350">
 
-## Using phishing resistant MFA options on mobile devices
+## Using phishing-resistant MFA options on mobile devices
 
-Using your mobile devices to access [company data](../computer-setup.md#company-data) is limited to just a few applications, such as Gmail, Calendar, Zoom, Slack, and Airbase. If you are going to be accessing any of these apps from your phone please log into the applications using phishing resistant methods.
+Using your mobile devices to access [company data](../computer-setup.md#company-data) is limited to a few applications, such as Gmail, Calendar, Zoom, Slack, and Airbase. If you are accessing any of these apps from your phone, please log into the applications using phishing-resistant methods.
 
 Once you add a Yubikey to your Okta account in the steps noted above, you will be able to use that Yubikey on your computer or phone as a form of mfa. You will not need to create a new registration entry per device, it is tied directly to your Okta account.
 
 1. Log into Okta with username and password
 1. Select 'Security Key or Biometric Authenticator'
 1. Select 'Verify'
-1. Tap and hold your key towards the tops of the device, if NFC compatible, or insert your Yubikey and tap the gold part
+1. Tap and hold your key towards the top of the device if NFC compatible, or insert your Yubikey and tap the gold part
 1. You will be logged in
 
-## I forgot my password/my login doesn't work, what do I do?
+## Enroll in MFA on a new laptop or desktop with a phishing-resistant factor
+
+When you attempt to enroll in MFA on another device, you may only use a security key or Okta FastPass to verify your identity on the other laptop or desktop. If you use a security key, it must be the same one used when enrolling on your first device.
+
+After you authenticate with your security key or Okta FastPass, Okta presents the other authenticators you can enroll in using your new device.
+
+If you have yet to enroll in a roaming authenticator (non-phishing resistant method), you may not be able to complete enrollment on other devices, depending on the device type.
+
+## Enroll in additional non-phishing-resistant factors
+
+You can enroll in other authenticators on your mobile device by going to the dashboard > settings > security settings. You will be prompted to authenticate with your phishing-resistant authenticator before you can add the additional authenticators.
```

Comment:
I would capitalize "...by going to the Dashboard > Settings > Security Settings..."